org.eclipse.richbeans.widgets.file/src/org/eclipse/richbeans/widgets/table/SeriesTable.java

```diff
@@ -260,6 +260,7 @@ public void setLockEditing(boolean checked) {
 	}
 
 	public List<ISeriesItemDescriptor> getSeriesItems() {
+		if (tableViewer==null) return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)